core/pkg/launch/job_builder.go

```diff
@@ -177,6 +177,7 @@ type JobBuilder struct {
 	wandbConfigParameters *launchWandbConfigParameters
 	configFiles           []*configFileParameter
 	saveShapeToMetadata   bool
+	inputSchema           *string
```

Comment:
Let's remove `inputSchema` and instead store the run config's input schema on the `wandbConfigParameters` attribute of `launchWandbConfigParameters`.